sections/homepage/Features/Features.tsx

```diff
@@ -181,58 +188,175 @@ const StyledFlexDivRow = styled(FlexDivRow)`
 	flex-wrap: wrap;
 	justify-content: center;
+
+	${media.lessThan('sm')`
+		flex-direction: column;
+		width: 305px;
+	`}
+`;
+
+const FeatureCard = styled(FlexDivRow)`
+	/* background: linear-gradient(180deg, rgba(40, 39, 39, 0.5) 0%, rgba(25, 24, 24, 0.5) 100%); */
+	/* box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25), inset 0px 1px 0px rgba(255, 255, 255, 0.1),
+	inset 0px 0px 20px rgba(255, 255, 255, 0.03); */
+	/* width: 403px; */
+	/* height: 135px; */
+	background-color: #1a1a1a;
+	border-radius: 10px;
+	padding: 30px;
+	border: 1px solid rgba(255, 255, 255, 0.05);
+	transition: all 0.2s ease-in-out;
+
+	:hover {
+		background-color: #202020;
+	}
+
+	${media.lessThan('sm')`
+		width: 305px;
+		height: auto;
+		align-items: center; 
+		padding: 20px 40px 20px 20px;
+	`}
 `;
 
 const IconGridContainer = styled(GridDivCentered)`
 	place-items: center;
 	justify-content: center;
-	grid-template-rows: 1fr 1fr;
-	grid-template-columns: repeat(4, 280px);
-	gap: 80px 20px;
-	margin-top: 80px;
-	svg {
-		width: 128px;
-		height: 64px;
+	grid-template-rows: 150px 150px;
+	grid-template-columns: repeat(4, 200px);
+	gap: 20px 20px;
+	margin-top: 40px;
+
+	.lyra {
+		svg {
+			width: 80px;
+		}
 	}
-`;
 
-const Container = styled.div`
-	margin: 140px 0px;
-`;
+	.synthetix {
+		svg {
+			width: 120px;
+		}
+	}
+	.aelin {
+		svg {
+			width: 95px;
+		}
+	}
 
-const FeatureCard = styled(FlexDivRow)`
-	background: linear-gradient(180deg, rgba(40, 39, 39, 0.5) 0%, rgba(25, 24, 24, 0.5) 100%);
-	box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25), inset 0px 1px 0px rgba(255, 255, 255, 0.1),
-		inset 0px 0px 20px rgba(255, 255, 255, 0.03);
-	border-radius: 15px;
-	padding: 32px 80px 32px 32px;
-	width: 403px;
-	height: 135px;
+	.thales {
+		svg {
+			width: 95px;
+		}
+	}
+
+	.graph {
+		svg {
+			width: 40px;
+		}
+	}
+
+	.hop {
+		svg {
+			width: 90px;
+		}
+	}
+
+	.chainlink {
+		svg {
+			width: 100px;
+		}
+	}
+
+	${media.lessThan('sm')`
+		grid-template-rows: repeat(4, 100px);
+		grid-template-columns: repeat(2, 140px);
+
+		.hop {
+			svg {
+				width: 70px;
+			}
+		}
+		.optimism {
+			svg {
+				width: 80px;
+			}
+		}
+		.aelin {
+			svg {
+				width: 80px;
+			}
+		}
+		.synthetix {
+			svg {
+				width: 85px;
+			}
+		}
+		.thales {
+			svg {
+				width: 80px;
+			}
+		}
+		.chainlink {
+			svg {
+				width: 85px;
+			}
+		}
+		.graph {
+			svg {
+				width: 35px;
+			}
+		}
+		.lyra {
+			svg {
+				width: 30px;
+			}
+		}
+	`}
 `;
 
 const PartnerIconContainer = styled.div`
-	place-self: center;
-	svg {
-		width: 128px;
-		height: 64px;
+	background-color: #1a1a1a;
+	border-radius: 8px;
+	place-items: center;
+	width: 100%;
+	height: 100%;
+	display: grid;
+	border: 1px solid rgba(255, 255, 255, 0.05);
+	transition: all 0.2s ease-in-out;
+
+	:hover {
+		background-color: #202020;
 	}
 `;
 
+const Container = styled.div`
+	margin: 140px 0px;
+`;
+
 const FeatureIconContainer = styled.div`
-	padding-bottom: 40px;
+	/* padding-bottom: 40px; */
```

Comment:
Seems like we might not need this anymore.

Reply:
Yes. Removed.